### ISSUE TITLE
Extension causes Sign In button to break

### ISSUE THREAD

**Issue by SpookyKipper** [clarkwinkelmann/flarum-ext-passwordless#15]
### Environment
```
Extension Version: 1.5.1 (via FreeFlarum)

Flarum 1.8.1
PHP 8.2.4
MySQL 10.11.2-MariaDB-1:10.11.2+maria~ubu2204-log
Laravel
8.83.27
```

### Error 
Freezes the whole website

### Console Error Message
```
TypeError: s.push is not a function
    at e.<anonymous> (index.ts:68:14)
    at forEach.t.<computed> [as footer] (extend.ts:39:16)
    at n.content (LogInModal.tsx:51:98)
    at n.view (Modal.tsx:164:19)
    at Function.a (render.js:30:16)
    at render.js:155:45
    at render.js:160:3
    at c (render.js:66:8)
    at l (render.js:50:5)
    at render.js:134:5
a @ mount-redraw.js:15
```

**Comment by clarkwinkelmann** [comment]
This appears to be caused by an undocumented breaking change in Flarum 1.8.

Only workaround is to disable the extension until I publish a fix. I'll try taking a look later today or tomorrow.

**Comment by clarkwinkelmann** [comment]
This should be fixed in version 1.5.2 of the extension. Not sure how often FreeFlarum updates them, it might be worth pinging them about it.
